script/Arcueid/Day 4/04_00_ARC04_1.txt

```diff
@@ -231,7 +231,7 @@ The time is still six in the morning.
 [sha:e8c05a7296b75eaec29bdcbf5c1a437a9e47aa59]{
 -- Page 20, Offset 5615.
 -- 何をするでもなく、20分間ぼんやりと部屋の天井を見上げ続けた。
-I stare up at the ceiling of my room for 20 minutes without doing anything.
+I stare up at the ceiling of my room for twenty minutes without doing anything.
```

Comment:
It's in number format in jp :/

Reply:
yeah but we're not following that

Reply:
I almost always prefer twenty to be written out. There are exceptions but as a general rule "it's like that in the jp" isn't a particularly solid argument.